_api_app/app/Sites/multisitesMenu.twig

```diff
@@ -0,0 +1,10 @@
+<ul id="multisites">
+  {% for site in sites %}
+      <li {{ site.isAvailable }} class="{{ site.isCurrentSite }}">
```

Comment:
`site.isAvailable` is not needed any more here. There should not be unavailable sites in `sites` array.

Reply:
`site.isCurrentSite` should probably be renamed to `site.className`. Probably we should check if this property is not empty otherwise we will get empty `class=""`